tests/cupy_tests/creation_tests/test_from_data.py

```diff
@@ -594,9 +594,10 @@ def __cuda_array_interface__(self):
         # synchronization is done via calling a cpdef function, which cannot
         # be mock-tested.
         if self.stream is not None:
-            # TODO(leofang): how about PTDS?
             if self.stream is cuda.Stream.null:
-                desc['stream'] = 1  # TODO(leofang): use runtime.streamLegacy
+                desc['stream'] = cuda.runtime.streamLegacy
+            elif (not cuda.runtime.is_hip) and self.stream is cuda.Stream.ptds:
+                desc['stream'] = cuda.runtime.streamPerThread
```

Comment:
This can be cleaned up too

Reply:
No, not here, because here we need to get the pointer from `self.stream`.